microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -212,36 +190,43 @@ If you have an offline deployment scenario, where app deployment has to work ent
 
 1. Include the Evergreen Standalone Installer in your app installer or updater.
 
-1. During your app setup, test whether the WebView2 Runtime is already installed, using either of the following approaches:
+1. During your app setup, run a test to make sure that the WebView2 Runtime is already installed. See [Detect if a suitable WebView2 Runtime is already installed](#detect-if-a-suitable-webview2-runtime-is-already-installed)
 
-   *  Inspect the `pv (REG_SZ)` regkey for the WebView2 Runtime at both of the following two locations.  The `HKEY_LOCAL_MACHINE` regkey indicates per-machine install, and the `HKEY_CURRENT_USE` regkey indicates per-user install. Either will suffice for WebView2 applications. If neither regkey exists, or if exists and is `null` or an empty string, this means that the WebView2 Runtime isn't installed on the client.  Use the regkeys to detect whether the WebView2 Runtime is installed, and to get the version of the WebView2 Runtime.  Find `pv (REG_SZ)` at the following locations:
+1. If the WebView2 Runtime isn't installed, run the Evergreen Standalone Installer.  If you want to run a silent installation, you can run the following command.
 
-      On 64-bit Windows:
+   <!-- keep the 3 instances or variants of this passage in sync: instance 3 -->
 
-      ```text
+   If you run the following command from an elevated process or command prompt, it triggers a _per-machine_ install.  If you don't run the command from an elevated process or command prompt, a _per-user_ install will take place.  However, a _per-user_ install is automatically replaced by a _per-machine_ install, if a _per-machine_ Microsoft Edge Updater is in place.  A _per-machine_ Microsoft Edge Updater is provided as part of Microsoft Edge, except for the Canary preview channel of Microsoft Edge.  For more information, see [Installing the Runtime as per-machine or per-user](#installing-the-runtime-as-per-machine-or-per-user).<!-- since this link is provided, the present paragraph could be shortened -->
+
+   ```Shell
+   MicrosoftEdgeWebView2RuntimeInstaller{X64/X86/ARM64}.exe /silent /install
+   ```
+
+#### Detect if a suitable WebView2 Runtime is already installed
+
+To verify that a WebView2 Runtime is installed, use either of the following approaches:
```

Comment:
I might say ", use one of the following methods:" not a show stopper just my 2 cents.